recipe/meta.yaml

```diff
@@ -16,6 +16,9 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
```

Comment:
Only numpy >= 1.19.2 currently has an installable package for  osx-arm64.
Probably we need to add (**not tested**):
```
    - numpy >= 1.19.2  # [osx-arm64 and build_platform != target_platform]
```

Reply:
I think you are right -- I'll try to keep an eye out for the new PR from the bot.